src/vdf_client.cpp

```diff
@@ -314,7 +314,7 @@ int gcd_128_max_iter=3;
 int main(int argc, char* argv[]) try
 {
     init_gmp();
-    if (argc != 4)
+    if (argc != 3)
```

Comment:
Thanks for the catch!

In this case, the message is wrong, but argc is correct. 

Chia blockchain calls the VDF process with 4 args:
https://github.com/Chia-Network/chia-blockchain/blob/main/chia/timelord/timelord_launcher.py#L54

so the usage would be `./vdf_client <host> <port> <proc_count>`

Reply:
I see, and I will create a new pull request to fix the wrong message and this one should be closed.